src/scss/06_components/navigation/_branding-navigation.scss

```diff
@@ -6,11 +6,11 @@ $-branding-icon-size-desktop: $spacing-double;
   background-color: $color-white;
   display: grid;
   grid-template-areas:
-    '. logo . brand search toggle .'
-    'menu menu menu menu menu menu menu'
-    'lang lang lang lang lang lang lang'
-    'langmenu langmenu langmenu langmenu langmenu langmenu langmenu';
-  grid-template-columns: var(--header-branding-padding) auto 1fr auto auto auto  var(--header-branding-padding);
+    '. logo . brand search toggle profile .'
```

Comment:
I think we should get rid of the whole `brand`-menu in drupal and css.

Reply:
I think we need to take this into a separate technical debt issue since I would need to do update hooks that remove the navigation element from drupal.

Reply:
Created an issue to Jira: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8006